frontend/src/pages/About.tsx

```diff
@@ -13,6 +13,7 @@ import dl from '../images/headshots/deyuan-li.jpg';
 import df from '../images/headshots/dylan-fernandez-de-lara.jpg';
 import eb from '../images/headshots/erik-boesen.jpg';
 import eh from '../images/headshots/evan-hu.jpg';
+import ff from '../images/headshots/filippo-fonseca.jpg';
```

Comment:
Could you move these changes out to a separate PR (will merge after merging this one)?

Reply:
done!